tests/model_serving/model_server/components/raw_deployment_serverless_co_exist/test_raw_deployment_serverless_inference_co_exist.py

```diff
@@ -57,11 +57,52 @@
     ],
     indirect=True,
 )
-class TestServerlessRawDeploymentInferenceCoExist:
-    def test_serverless_openvino_created_before_raw_deployment_caikit_inference(
+class TestServerlessRawInternalDeploymentInferenceCoExist:
+    def test_serverless_openvino_created_before_raw_internal_deployment_caikit_inference(
```

Comment:
This is using the same fixtures as test_raw_internal_deployment_caikit_created_after_serverless_in_namespace_rest_inference(). Can these two be parameterized now?

Reply:
preferred to split (i think it is because of https://github.com/pytest-dev/pytest/issues/9287) - it was a mess trying to understand what is going on in the logs.
so prefer to have different classes where the logs are more readble